libgrive/src/base/Drive.hh

```diff
@@ -41,18 +41,19 @@ public :
 	Drive( Syncer *syncer, const Val& options ) ;
 
 	void DetectChanges() ;
+	bool Move();
```

Comment:
And here I think Drive::Move() should take file arguments instead of taking them from Config object...